bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/jsons/JsonAnnouncement.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.amazonechocontrol.internal.jsons;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.amazonechocontrol.internal.jsons.JsonDevices.Device;
+
+/**
+ *
+ * @author Tom Blum (Trinitus01) - Initial contribution
+ */
+@NonNullByDefault
+public class JsonAnnouncement {
+
```

Comment:
As far as I understand is this class a helper class for the queue. You should not put them in the jsons folder and you should not name it with Json prefix. All the jsons in the folder a the real REST API Jsons of amazon.

Reply:
Ok, else if i need to convert it to json to collect and merge the incomming announcements to only get one request instead of x requests? It is not for the REST API but one of the main parts is to convert it to a json.

Reply:
> 
> 
> @Trinitus01 Thank you for helping to improve the binding. One optimization would maybe make sense and would be possible now with the queue. In the request for announcements can you specifiy a array of devices. So if we have multiple Announcements with the same content and the same volume, it would be possible to send them with only one request. The sound output would be in this case very synchrone.

Maybe i misunderstand your comment but announcements with the same content are send with only one request at the moment. So i do not understand what to change. Maybe you point on the difference between tts and act, that its necessary to use one device which contains multiple deviceids? This is a very strange structure of amazons act json.

Reply:
Oh, sorry, have not seen that is should work already in this way. perfect!